fishery-bulletin.csl

```diff
@@ -0,0 +1,377 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-US">
+  <info>
+    <title>Fishery Bulletin</title>
+    <id>http://www.zotero.org/styles/fishery-bulletin</id>
+    <link href="http://www.zotero.org/styles/fishery-bulletin" rel="self"/>
+    <link href="http://www.zotero.org/styles/behaviour" rel="template"/>
+    <link href="http://fishbull.noaa.gov/fguide.htm" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <issn>0090-0656</issn>
+    <updated>2017-10-06T10:33:53+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="editortranslator" form="short">
+        <single>ed. &amp; trans.</single>
+        <multiple>eds. &amp; trans.</multiple>
+      </term>
+      <term name="translator" form="short">
+        <single>trans.</single>
+        <multiple>trans.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="container-contributors">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <names variable="editor translator" delimiter=", " prefix=" (" suffix=").">
+          <name and="text" delimiter-precedes-last="always" initialize-with="." name-as-sort-order="first"/>
+          <label form="short" strip-periods="true" prefix=", "/>
+        </names>
+      </if>
+    </choose>
+  </macro>
+  <macro name="secondary-contributors">
+    <choose>
+      <if type="chapter paper-conference" match="none">
+        <names variable="translator editor" delimiter=", " prefix=" (" suffix=")">
+          <name and="symbol" initialize-with=". " delimiter=", "/>
+          <label form="short" prefix=", " text-case="title"/>
```

Comment:
you're sure the labels for editors are different for chapters (above: lowercase, stripped periods) and books (uppercase, include periods)? Absent guidance, I'd unify this.